packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -507,6 +507,7 @@ contract LockedGold is
     }
 
     _updateDelegatedAmount(account);
+    _updateDelegatedAmount(reporter);
```

Comment:
If I remember correctly, the whole point of having an `updateDelegatedAmount` function was that we'd allow the stored delegated amount to diverge from the expected amount (in particular, when the delegator accrues epoch rewards). So this change isn't even strictly necessary, more of a convenience that adds gas costs to the `slash` function. 

In fact, even the above call with `_updateDelegatedAmount(account)` isn't really necessary - since the only slashable actors are validators and validator groups, and those are not allowed to delegate.